module #2. REST API Basics/rest-api-basic/core/src/main/java/com/epam/esm/configuration/servlet/DispatcherConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.configuration.servlet;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.epam.esm")
+public class DispatcherConfig implements WebMvcConfigurer {
```

Comment:
Maybe this configuration is not needed at all if it is needed please leave a java doc of purpose of this configuration